cmd/git-sync/main.go

```diff
@@ -64,6 +66,10 @@ var flSyncTimeout = flag.Int("timeout", envInt("GIT_SYNC_TIMEOUT", 120),
 	"the max number of seconds for a complete sync")
 var flOneTime = flag.Bool("one-time", envBool("GIT_SYNC_ONE_TIME", false),
 	"exit after the initial checkout")
+var flSpecific = flag.Bool("specific", envBool("GIT_SYNC_SPECIFIC", false),
+	"checkout at a specific time daily")
+var flSpecificTime = flag.String("specific-time", envString("GIT_SYNC_SPECIFIC_TIME", "0000"),
```

Comment:
Lets use 1 flag.

In your logic just check if its not nil and that will tell you if they want to use sync time. I think you can make the default in `envString` be nil instead of 0000